src/Operation/Aggregate.php

```diff
@@ -337,11 +298,9 @@ private function createCommandDocument(): array
             $cmd['hint'] = is_array($this->options['hint']) ? (object) $this->options['hint'] : $this->options['hint'];
         }
 
-        if ($this->options['useCursor']) {
-            $cmd['cursor'] = isset($this->options["batchSize"])
-                ? ['batchSize' => $this->options["batchSize"]]
-                : new stdClass();
-        }
+        $cmd['cursor'] = isset($this->options["batchSize"])
+            ? ['batchSize' => $this->options["batchSize"]]
+            : new stdClass();
```

Comment:
I assume always supplying a `cursor` option for the `explain` option is kosher?

Quoting [Aggregate: Example](https://www.mongodb.com/docs/manual/reference/command/aggregate/#example) from the server docs:

> MongoDB 3.6 removes the use of [aggregate](https://www.mongodb.com/docs/manual/reference/command/aggregate/#mongodb-dbcommand-dbcmd.aggregate)  command without the cursor option unless the command includes the explain option. Unless you include the explain option, you must specify the cursor option.

It doesn't say anything about `explain` _prohibiting_ a `cursor` option, but `cursor` does seem to be ignored. Do you think we should gate this with a `! $this->isExplain` condition?

Reply:
I'd leave the option in unless `explain` specifically prohibits it - this is consistent with other commands where we also send `batchSize` as part of an `explain`.